src/components/Layouts/Primary/TopBar/Items/SearchMenu/SearchDialog.tsx

```diff
@@ -172,64 +182,70 @@ export const SearchDialog: React.FC<SearchDialogProps> = ({ handleClose }) => {
       link: `/accountLists/${accountListId}/tools`,
     },
     {
-      name: t('Coaching'),
+      name: t('Tools - Appeals'),
       icon: <CompassIcon />,
-      link: `/accountLists/${accountListId}/coaching`,
+      link: `/accountLists/${accountListId}/tools`,
```

Comment:
This is incorrect. it needs to go to 
```suggestion
      link: `/accountLists/${accountListId}/tools/appeals`,
```

Also, I'm unsure how this has happened, but the Tools page looks like the Blue Man music group have taken over.
![Screenshot 2025-04-07 at 3 15 56 PM](https://github.com/user-attachments/assets/2fcca1df-d22f-42d7-a533-bca173277f74)

Reply:
![image](https://github.com/user-attachments/assets/b630e201-818e-46bf-bc3f-f395106f1748)

Reply:
Thanks for catching the incorrect URL